src/common.props

```diff
@@ -15,8 +15,8 @@
 
   <!-- Version settings: https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/ -->
   <PropertyGroup>
-    <MajorVersion>1</MajorVersion>
-    <MinorVersion>4</MinorVersion>
+    <MajorVersion>2</MajorVersion>
+    <MinorVersion>0</MinorVersion>
```

Comment:
@cgillum , I’m wondering if it might make more sense to consider a major version upgrade instead of a minor one, considering we have major version upgrades for both DurableTask.Core and WebJobs.Extensions.DurableTask.

I have a similar PR for Netheriteb https://github.com/microsoft/durabletask-netherite/pull/437, and Sebastian and David both prefer a major version release. Thus, I think I should apply this to mssql  as well.  Let me know your thoughts on this. Thanks!

Reply:
I think I'd rather not make a major version change for MSSQL. There shouldn't be any breaking dependency or behavior changes, so I feel like doing a major version increment is a bit misleading. Semantic versioning AFAIK doesn't make any recommendation about how to handle major version changes for dependencies, so I think it's up to the library owner's discretion. More discussion on this [here](https://softwareengineering.stackexchange.com/a/168661). I'm okay if MSSQL uses a different versioning strategy than Netherite.